config/quick-start/worker/proxmox-machine-template-worker.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: ProxmoxMachine
```

Comment:
and this should be ProxmoxMachineTemplate too.
then you can just move `spec` field to `spec.template.spec`